ArmState rArm
ArmState lArm
float64 rArmVelocity # Unused 
float64 lArmVelocity # Unused
